### PR TITLE
[NO-TICKET] Color token updates for Core, HCgov and Mgov

### PR DESCRIPTION
**Pull request by line47**
- Updating color values for Core, HCgov and Mgov
- Correcting color values or updating color swatches to be pulled from the same color set

### CODE REVIEW

**Comment by line47** [comment]
Hey @forrestbaer - Maybe we can tag up sometime today so you can show me what I need to do to get this PS merged and updated

**Comment by forrestbaer** [comment]
Looks good, in the future you'll run 

`yarn token:dist` from the root DS directory which will output the sass/css changes based on these updates, but for this PR run:

`yarn --cwd ./packages/design-system-tokens build themes scss && yarn --cwd ./packages/design-system-tokens build themes css-vars && yarn --cwd ./packages/design-system-tokens dist`

Which is what `yarn token:dist` will be when my (#1948) PR is merged.

When that's done, check in those changes and can continue!

**Comment by line47** [comment]
@forrestbaer Awesome, thank you! I rand the commands and pushed up those changes.   
Is there a place i can preview these changes on a doc site or in storybook?

File: packages/ds-medicare-gov/src/styles/settings/variables/_medicare-defaultTheme.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
+$color-teal-700: #0e4a41;
```
Comment (forrestbaer) [inline packages/ds-medicare-gov/src/styles/settings/variables/_medicare-defaultTheme.scss:1]: Hmm this looks like an old file, maybe was leftover from a previous run of this command? This shouldn't be here and shouldn't have been created by the command I sent you.  Can remove this?

File: packages/ds-healthcare-gov/src/styles/settings/_healthcare-defaultTheme.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+$color-primary: #046791;
```
Comment (forrestbaer) [inline packages/ds-healthcare-gov/src/styles/settings/_healthcare-defaultTheme.scss:1]: This file should be removed, was created long ago and was likely just hanging out in your filesystem :)

File: packages/design-system/src/styles/settings/variables/_core-defaultTheme.scss [hunk 3]
```diff
@@ -0,0 +1,145 @@
+$animation-ease-in-out-expo: cubic-bezier(1, 0, 0, 1) !default;
```
Comment (forrestbaer) [inline packages/design-system/src/styles/settings/variables/_core-defaultTheme.scss:1]: This one as well can be removed.

**Review by forrestbaer — COMMENTED** [review]
Couple comments, looks like you had some files in there from a while back.

File: packages/design-system/src/styles/settings/variables/_core-defaultTheme.scss (hunk 3 above)
Comment (line47) [reply packages/design-system/src/styles/settings/variables/_core-defaultTheme.scss:1]: Nevermind I see the other theme files, I'll remove these :+1:

File: .storybook/_core-theme.css [hunk 4]
```diff
@@ -68,7 +68,7 @@
 --color-primary-alt: #02bfe7;
```
[the remark this answers is not in the document]
Comment (forrestbaer) [reply .storybook/_core-theme.css:68]: from the ds root: `git rm .storybook/_*.css && git commit -m 'removing some files' && git push` if you want a quick cut/paste :)

[the remark this answers is not in the document]
Comment (line47) [reply .storybook/_core-theme.css]: Thanks @forrestbaer I think i got them removed :)

**Review by forrestbaer — APPROVED** [review]

**Pull request merged**